hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/src/main/native/src/NativeTask.h

```diff
@@ -59,8 +59,8 @@ enum Endium {
 #define NATIVE_HADOOP_VERSION "native.hadoop.version"
 
 #define NATIVE_INPUT_SPLIT "native.input.split"
-#define INPUT_LINE_KV_SEPERATOR "mapreduce.input.keyvaluelinerecordreader.key.value.separator"
```

Comment:
leave these as they were before. Yes, its wrong, but changing it will break everything which compiles against it

Reply:
Thanks! Leave these, Have one more commit to fix it!